bitnami/rabbitmq/README.md

```diff
@@ -284,192 +284,82 @@ $ helm install my-release -f values.yaml bitnami/rabbitmq
-            "tags": "administrator"
-          }
-        ],
-        "vhosts": [
-          {
-            "name": "/"
-          }
-        ]
-      }
-loadDefinition:
-  enabled: true
-  existingSecret: load-definition
-extraConfiguration: |
-  load_definitions = /app/load_definition.json
-```
+Refer to the chart documentation for [more information and configuration examples of loading custom definitions](https://docs.bitnami.com/kubernetes/infrastructure/rabbitmq/configuration/load-files/).
 
-### LDAP
+### Configure LDAP support
 
-LDAP support can be enabled in the chart by specifying the `ldap.` parameters while creating a release. The following parameters should be configured to properly enable the LDAP support in the chart.
+LDAP support can be enabled in the chart by specifying the `ldap.*` parameters while creating a release. The following parameters should be configured to properly enable the LDAP support in the chart.
 
 - `ldap.enabled`: Enable LDAP support. Defaults to `false`.
 - `ldap.servers`: List of LDAP servers hostnames. No defaults.
 - `ldap.port`: LDAP servers port. `389`.
 - `ldap.user_dn_pattern`: Pattern used to translate the provided username into a value to be used for the LDAP bind. Defaults to `cn=${username},dc=example,dc=org`.
 - `ldap.tls.enabled`: Enable TLS for LDAP connections. Defaults to `false`.
 
-For example:
+Refer to the chart documentation for [more information and a configuration example](https://docs.bitnami.com/kubernetes/infrastructure/rabbitmq/configuration/configure-ldap/).
 
-```console
-ldap.enabled=true
-ldap.serverss[0]="my-ldap-server"
-ldap.port="389"
-ldap.user_dn_pattern="cn=${username},dc=example,dc=org"
-```
-
-If `ldap.tls.enabled` is set to true, consider using `ldap.port=636` and checking the settings in the advancedConfiguration.
+### Configure memory high watermark
 
-### Memory high watermark
+It is possible to configure a memory high watermark on RabbitMQ to define [memory thresholds](https://www.rabbitmq.com/memory.html#threshold) using the `memoryHighWatermark.*` parameters. To do so, you have two alternatives:
 
-It is possible to configure Memory high watermark on RabbitMQ to define [memory thresholds](https://www.rabbitmq.com/memory.html#threshold) using the `memoryHighWatermark.*` parameters. To do so, you have two alternatives:
+* Set an absolute limit of RAM to be used on each RabbitMQ node.
+* Set a relative limit of RAM to be used on each RabbitMQ node. To enable this feature, you must define the memory limits at pod level too.
 
-A) Set an absolute limit of RAM to be used on each RabbitMQ node:
-
-```console
-memoryHighWatermark.enabled="true"
-memoryHighWatermark.type="absolute"
-memoryHighWatermark.value="512MB"
-```
+Refer to the chart documentation for [more information and configuration examples of defining a memory high watermark](https://docs.bitnami.com/kubernetes/infrastructure/rabbitmq/configuration/configure-memory/).
```

Comment:
In this case, as the doc is pretty short and just include a list of parameters, maybe we can leave that doc into the readme. No strong though but it looks to me there is no info in the readme and we are forcing the user anoter link with just small info.

Reply:
OK done